website/docs/Software/OpenBCISoftware/01-OpenBCI_GUI.md

```diff
@@ -319,8 +319,69 @@ If you are encountering issues launching the GUI at this point, please head to t
 
 ![GUI Screenshot convert large SD recording](../../assets/SoftwareImages/OpenBCISoftware/gui_convert_large_SD_recordings_scrnshot.png)
 
-## Cyton Other Channels
-
+## Exported Data
+
+Data from the board will be recorded into a **Brainflow-RAW_date_time** CSV file and **OpenBCI-RAW_date_time** CSV file in txt format. All data saved to these files are unfiltered. These recordings are stored in ```User/Documents/OpenBCI_GUI/Recordings```. Each GUI session will have its own directory titled **OpenBCISession_date_time**. Every time you start and stop a session creates a new Brainflow-RAW and OpenBCI-RAW file within that session folder. The picture below shows three started and stopped recordings within one GUI session.
+
+![GUI data files](../../assets/SoftwareImages/OpenBCISoftware/GUI_data_files.png)
+
+### Cyton
+
+Here is a table of the column headers and descriptions for the Cyton.
+
+| Column | Name                    | Description |
+| ------ | ----------------------- | ------------- |
+| 1      | Sample Index            | The index of the sample per second (0-250) |
+| 2      | EXG Channel 0           | EEG/EMG/ECG channel connected to pin labeled N1P         |
```

Comment:
This description isn't entirely accurate, in that each channel can be affected by the "pins" labeled N1P. There is an N pin and a P pin for each channel.